pkg/instance/tshark.go

```diff
@@ -0,0 +1,54 @@
+package instance
+
+import (
+	"context"
+)
+
+const (
+	tsharkCollectorName       = "tshark-collector"
+	tsharkCollectorImage      = "ghcr.io/celestiaorg/tshark-s3:pr-5"
+	tsharkCollectorCPU        = "100m"
+	tsharkCollectorMemory     = "250Mi"
+	tsharkCollectorVolumePath = "/tshark"
+	netAdminCapability        = "NET_ADMIN"
+)
+
+func (i *Instance) createTsharkCollectorInstance(ctx context.Context) (*Instance, error) {
+	tsharkCollector, err := New(tsharkCollectorName, i.SystemDependencies)
+	if err != nil {
+		return nil, err
+	}
+	if tsharkCollector.SetImage(ctx, tsharkCollectorImage) != nil {
+		return nil, err
+	}
+	if tsharkCollector.Commit() != nil {
+		return nil, err
+	}
+	if tsharkCollector.SetCPU(tsharkCollectorCPU) != nil {
+		return nil, err
+	}
+	if tsharkCollector.SetMemory(tsharkCollectorMemory, tsharkCollectorMemory) != nil {
+		return nil, err
+	}
+	if tsharkCollector.AddVolume(tsharkCollectorVolumePath, i.tsharkCollectorConfig.volumeSize) != nil {
```

Comment:
Would also be nice to refactor these function to receive resource.Quantity type instead. We can do it in a follow up PR.

Reply:
issue created #426